pkg/operator/ceph/object/admin.go

```diff
@@ -18,13 +18,19 @@ package object
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/pkg/errors"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/daemon/ceph/client"
 )
 
+const (
+	//RGWCmdTimeout is the timeout used when running rgw cli commands
+	RGWCmdTimeout = 2 * time.Minute
```

Comment:
```suggestion
	rgwCmdTimeout = 2 * time.Minute
```

I don't think we need to export it, right?

Reply:
yes. Fixed.